src/components/title.tsx

```diff
@@ -0,0 +1,35 @@
+import * as React from 'react'
+import styled from 'styled-components'
+import * as fonts from '../assets/fonts'
+
+type TitleSize = "small" | "medium" | "large" | "xlarge" | "xxlarge"
+
+interface TitleProps extends React.HTMLProps<JSX.Element> {
+  titleSize?: TitleSize
+}
+
+const titleSizes = {
+  small: '25px',
+  medium: '30px',
+  large: '37px',
+  xlarge: '50px',
+  xxlarge: '72px'
+}
+
+const Title: React.SFC<TitleProps> = props => (
+  <div className={props.className}>
```

Comment:
Good stuff, was wondering about this too 👍 

@craigspaeth Do you have any opinion on what to use going forward?